src/client/driver/command-executors/client-functions/selector-executor/index.js

```diff
@@ -1,28 +1,13 @@
 import { Promise } from '../../../deps/hammerhead';
-import { delay, positionUtils, domUtils } from '../../../deps/testcafe-core';
-import { selectElement as selectElementUI } from '../../../deps/testcafe-ui';
+import { delay } from '../../../deps/testcafe-core';
 import ClientFunctionExecutor from '../client-function-executor';
+import { exists, visible } from '../visibility-function';
```

Comment:
Maybe it's better to rename `visibility-function.js` to something like `element-constraints.js`? \cc @AlexanderMoskovkin

Reply:
I doubt about the `constraints` word but I agree with the idea to rename this since `exists` is not `visibility function`. 
Maybe something like `check-element` or `element-utils`